hasura/metadata/actions.graphql

```diff
@@ -1,89 +1,135 @@
 type Mutation {
-  adminUpdateUser(payload: AdminUpdateUserInput!): UserResponse
+  adminUpdateUser(
```

Comment:
can it be because of your hasura-cli version difference hence the formatting changes ?

Reply:
I'm not sure where these differences in formatting come from. I'm using `yarn hasura console` so it's using hasura-cli 2.10.1. Hasura docker image is also the same. The only thing that might differ is my OS - I'm on Ubuntu 20.04